### DISCUSSION TITLE
I found that using comfyui in swarm will conflict with the plug-in in comfyui that will pop up a new selection window.

### DISCUSSION THREAD

**Discussion by 1025307336** [mcmonkeyprojects/SwarmUI#190] [General]
Such as: https://github.com/AIGODLIKE/AIGODLIKE-ComfyUI-Studio

Will cause the comfy tab to have a white screen directly

Maybe it can be put into the swarmui introduction to remind users

**Comment by mcmonkey4eva** [comment]
Nothing should break - if something like that does, it means there's incorrect code in that extension.
